lang/es.yml

```
@@ -0,0 +1,103 @@
+WELCOME: "Bienvenido a {0}"
+APP_NAME: "Legendario Fácil"
+UNCOMPILED_VERSION_DETECTED: "Se detectó una versión decompilada de {0}."
+UNSTABLE_BUILD_WARNING: "Este artifacto puede llegar a ser inestable."
+SELECT_VALID_OPTION: "¡Selecciona una opción válida!"
+EMPTY_PLACEHOLDER: "¡Nada por aquí!"
+SELECT_ACTION: "Selecciona qué quieres hacer:"
+
+ACTIONS:
+    START: "iniciar"
+    IMPORT: "importar"
+    IMPORT_GAME_IN_PATH: 'importar el juego "{0}", localizado en "{1}"'
+    INSTALL: "instalar"
+    INSTALL_GAME_IN_PATH: 'instalar "{0}" en "{1}"'
+    MOVE_GAME_TO_PATH: 'mover el juego "{0}" hacia "{1}"'
+    UNINSTALL: "desinstalar"
+    UNINSTALL_GAME: "desinstalar {0}"
+    UPDATE: "actualizar"
+    UPDATE_GAME: "actualizar {0}"
+    MOVE: "mover"
+    VERIFY: "verfificar los archivos"
+    VERIFY_GAME: 'verificar los archivos de "{0}"'
+    FIND: "encontrar los archivos de"
+    SHORTCUT: "hacer un acceso directo para"
+    INFO: "ver información para"
+    UPDATE_TO_VERSION: "actualizar a la versión {0}"
+    SYNC_GAMES: "sincronizar los juegos con el Lanzador de Epic Games"
+    SYNC_ONLINE_SAVES: "sincronizar tus partidas guardadas en la nube"
+    DELETE_CORRUPTED_ONLINE_SAVES: "borrar las partidas corrompidas de la nube"
+
+LIST_OWNED_GAMES: "Catalogar los juegos de los cuales eres dueño"
+LIST_INSTALLED_GAMES: "Catalogar los juegos instalados o importados"
+START_GAME: "Iniciar un juego para la posterior ejecución del mismo"
+INSTALL_GAME: "Instalar un juego"
+UPDATE_GAME: "Actualizar un juego"
+IMPORT_GAME: "Importar un juego instalado"
+UNINSTALL_GAME: "Desinstalar un juego"
+BROWSE_GAME_FILES: "Navegar a través de los archivos de un juego"
+SHOW_GAME_INFO: "Ver la información acerca de un juego"
+SYNC_CLOUD_SAVES: "Sincronizar las partidas guardadas con la nube"
+SYNC_WITH_EGL: "Sincronizar los juegos con el Lanzador de Epic Games"
+VERIFY_GAME_FILES: "Verificar la integridad de los archivos de un juego"
+MOVE_GAME: "Mover un juego a otra carpeta en el disco"
+CREATE_SHORTCUT: "Crear un acceso directo para un juego"
+REFRESH_CACHE: "Refrescar o actualizar el búfer de la memoria intermedia"
+CHECK_FOR_UPDATES: "Comprobar si hay actualizaciones nuevas para la aplicación"
+EXIT: "Salir de la aplicación"
+
+PRESS_ENTER_TO_RETURN_TO_MAIN_MENU: "Presiona enter para regresar al menú principal"
+SELECT_THIS_ITEM_TO_EXIT: "Selecciona este item para salir..."
+SAVING_CACHE: "Guardando la memoria intermedia en el disco..."
+GOODBYE: "¡Adiós!"
+TYPE_A_VALID_PATH: "Por favor escribe una ruta válida"
+GAME_NOT_FOUND: "¡Juego no encontrado!"
+
+FILE_EXPLORER_WINDOW: "Una ventana del explorador de Windows se debería haber abierto en la ubicación de los archivos juego."
+UP_TO_DATE: "¡No hay ninguna actualización disponible! ¡Estás actualizado!"
+UPDATE_AVAILABLE: "¡Hay una actualización disponible!"
+CHANGELOG: "Registro de cambios"
+USING_UNCOMPILED_VERSION: "Estás usando una versión no compilada de Fácil Legendario."

```

```suggestion
USING_UNCOMPILED_VERSION: "Estás usando una versión no compilada de {0}."
```
Should be replaced with {0} and then replace {0} in the code with the localized APP_NAME text.